digital_image_processing/sepia.py

```diff
@@ -0,0 +1,54 @@
+"""
+    Implemented an algorithm using opencv to tone an image with sepia technique
+"""
+
+from cv2 import imread, imshow, waitKey, destroyAllWindows
+
+
+def make_sepia(img, factor: int):
+    """ Function create sepia tone. Source: https://en.wikipedia.org/wiki/Sepia_(color) """
+    pixel_h, pixel_v = img.shape[0], img.shape[1]
+
+    def to_grayscale(blue, green, red):
+        """
+        Helper function to create pixel's greyscale representation
+        Src: https://pl.wikipedia.org/wiki/YUV
+        """
+        return 0.2126 * red + 0.587 * green + 0.114 * blue
+
+    def normalize(value):
+        """ Helper function to normalize R/G/B value -> return 255 if value > 255"""
+        return value if value <= 255 else 255
```

Comment:
```suggestion
        return min(value, 255)
```
https://docs.python.org/3/library/functions.html#min